src/charm.py

```diff
@@ -25,7 +25,7 @@
 from ops.pebble import PathError
 from requests import Session
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
+from requests.packages.urllib3.util.retry import Retry  # type: ignore
```

Comment:
I'd put this kind of stuff in pyproject
https://github.com/canonical/grafana-agent-k8s-operator/blob/bcb44b685c21f319b5621eb8d0bd52d0d3e9ec89/pyproject.toml#L54-L55

Reply:
Done